tools.go

```diff
@@ -0,0 +1,8 @@
+package tools
```

Comment:
I'm not a fan of "tools", feels like "util" and we could add other Go files here later.

```suggestion
package frsca
```